usecases/replica/replicator.go

```diff
@@ -101,6 +116,22 @@ func (r *Replicator) DeleteObject(ctx context.Context, localhost, shard string,
 	return coord.Replicate(ctx, op, r.simpleCommit())
 }
 
+func (r *Replicator) DeleteObjects(ctx context.Context, localhost, shard string,
```

Comment:
Previous implementations  (e.g. db.index) of DeleteObjects return []BatchSimpleObject ({UUID, ERROR}).
I don't think we need to return UUID because I supposed UUIDs are contained in the request itself. Therefore, I approved it.